packages/zoe/src/cleanProposal.js

```diff
@@ -100,62 +157,27 @@ export const cleanKeywords = keywordRecord => {
  * `{ waived: null }` `{ onDemand: null }` `{ afterDeadline: { timer
  * :Timer, deadline :bigint } }
  *
- * @param {(brand: Brand) => DeprecatedAmountMath} getAmountMath
  * @param {Proposal} proposal
  * @returns {ProposalRecord}
  */
-export const cleanProposal = (getAmountMath, proposal) => {
+export const cleanProposal = proposal => {
```

Comment:
I keep referring to our patterns as "amount patterns". I should stop. They are more general patterns over Comparables. Once we have those, I imagine that a lot of these kinds of structure validations can be expressed much more intuitively, more reliably, and much tinier by writing patterns that do most of that validation by matching. No change suggested until then.